### ISSUE TITLE
MathML span position

### ISSUE THREAD

**Issue by pd4d10** [KaTeX/KaTeX#2796]
> Before reporting a bug
- [x] Check [common issues](https://katex.org/docs/issues.html).
- [x] Check the bug is reproducible in [the demo](https://katex.org). If not, check KaTeX is up-to-date and installed correctly.
- [x] Search for [existing issues](https://github.com/KaTeX/KaTeX/issues).

**Describe the bug:**

The MathML span's position seems weird in our [Markdown editor component](https://github.com/bytedance/bytemd). Demo here: https://bytemd.netlify.app/

**(La)TeX code:**
The code of (La)TeX you tried to render:
```latex
\displaystyle \left( \sum_{k=1}^n a_k b_k \right)^2 \leq \left( \sum_{k=1}^n a_k^2 \right) \left( \sum_{k=1}^n b_k^2 \right)
```

**Screenshots:**

![image](https://user-images.githubusercontent.com/9524411/108308593-f284af80-71ea-11eb-820a-a36791f48007.png)

**Environment (please complete the following information):**
 - KaTeX Version: v0.12.0
 - Device: Desktop
 - OS: macOS big sur
 - Browser: Chrome
 - Version: 88.0.4324.182

**Comment by mattford** [comment]
Not involved with this project but I had a quick look at this example out of interest and found that if I remove the newlines before/after the `$$`'s, it displays left aligned:

```
$$\displaystyle \left( \sum_{k=1}^n a_k b_k \right)^2 \leq \left( \sum_{k=1}^n a_k^2 \right) \left( \sum_{k=1}^n b_k^2 \right)$$
```

**Comment by ronkok** [comment]
In KaTeX default configuration, the math is supplied twice: (1) In HTML, to be visually rendered by the browser. (2) In MathML, for accessibility (screen reader). The MathML is not intended to appear visually.

**Comment by pd4d10** [comment]
> The MathML is not intended to appear visually.

Yep, noticed that the MathML node size is 1x1. But in this case, its position seems weird (see the screenshot)

**Comment by eikowagenknecht** [comment]
Had the same problem today. This happens when the parent elements are `position: static` (which is the default when no position is fixed) and can be fixed by setting `position: relative` on the element containing the KateX rendering.

Also adding `left: -10000px;` to `.katex .katex-mathml` is helpful:

```css
.katex .katex-mathml {
  left: -10000px;
}
```
